src/test/java/racingCar/RacingGame.java

```diff
@@ -0,0 +1,28 @@
+package racingCar;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingGame {
+  private int tryCount;
+  private final List<Car> cars;
+
+  public RacingGame(int carNumber, int tryCount) {
+    cars = new ArrayList<>();
+    for (int i=0; i < carNumber ; i++) {
```

Comment:
```suggestion
    for (int i = 0; i < carNumber; i++) {
```
ide 에서 제공되는 reformat 기능을 이용하면 한번에 표준 format 으로 맞추실 수 있습니다!